src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -105,6 +101,8 @@ private constructor(
             displayText = "",
         )
     addMessageAtIndex(responsePlaceholder, index = messages.count(), shouldAddBlinkingCursor = true)
+
+    submitMessageToAgent(humanMessage, contextFiles)
```

Comment:
I just moved it from line 94 - it makes more sense to call it after we add the placeholder